libs/sdk-ui-ext/src/internal/dashboardEmbedding/DashboardLayout/utils/sizing.ts

```diff
@@ -376,13 +438,15 @@ export const getResizedItemPositions = <TWidget>(
         }, positions);
 };
 
-export const getDashboardLayoutItemHeight = (size: IDashboardLayoutSize, screen: ScreenSize) => {
-    const { heightAsRatio, gridHeight } = size;
-    if (gridHeight) {
+export const getDashboardLayoutHeight = (
+    size: IDashboardLayoutSize,
+    enableCustomHeight: boolean,
+): number | undefined => {
+    const { gridHeight } = size;
+    if (gridHeight && enableCustomHeight) {
```

Comment:
is this needed if we can remove gridHeights completely before sending them to render?